### PR TITLE
feat(unlock-app): Use different qr scanner library for verification page

### PR DESCRIPTION
**Pull request by searchableguy**
# Description

The underlying library behind react-qr-reader (zxing) doesn't work properly with some of our generated QR code. I've replaced it with qr-scanner package which uses jsqr library for qrcode scanning which seems to have better success rate in my testing.

# Issues

Fixes #
Refs #

# Checklist:

- [ ] 1 PR, 1 purpose: my Pull Request applies to a single purpose
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have updated the docs to reflect my changes if applicable
- [ ] I have added tests (and stories for frontend components) that prove my fix is effective or that my feature works
- [ ] I have performed a self-review of my own code
- [ ] If my code involves visual changes, I am adding applicable screenshots to this thread

### CODE REVIEW

File: unlock-app/src/utils/verification.ts [hunk 1]
```diff
@@ -3,7 +3,9 @@ import * as z from 'zod'
 export const MembershipVerificationData = z.object({
   account: z.string(),
   timestamp: z.number(),
-  tokenId: z.string(),
+  tokenId: z
+    .union([z.string(), z.number()])
+    .transform((value) => value.toString()),
```
Comment (searchableguy) [inline unlock-app/src/utils/verification.ts:8]: Different data type in QR codes.

File: unlock-app/src/components/interface/VerificationStatus.tsx [hunk 2]
```diff
@@ -127,7 +143,9 @@ export const VerificationStatus = ({ config }: Props) => {
   )
 
   const invalid = invalidMembership({
-    membershipData: membershipData!,
+    keyId: key!.tokenId.toString(),
+    owner: key!.owner,
+    expiration: key!.expiration,
```
Comment (searchableguy) [inline unlock-app/src/components/interface/VerificationStatus.tsx:148]: Same here. We don't consistent data so I'm directly passing the props after converting.

**Comment by julien51** [comment]
This needs very thorough testing. Have you been able to do some of this?

**Comment by searchableguy** [comment]
Tests done

[ ] Disconnect and scan - make sure it works without logging in
[ ] Able to check in ticket as a verifier
[ ] Able to check in ticket as a lock manager
[ ] Checked in ticket shows as yellow with correct time
[ ] Test with v10 and v9 locks respectively.
[ ] Use the connect button to login

**Review by kalidiagne — APPROVED** [review]

**Review by julien51 — APPROVED** [review]

**Pull request merged**